docs/dev/wasm.md

```diff
@@ -50,75 +50,69 @@ and passing for each parameter a 64-bit value, of which top 32 bits are its size
 }
 ```
 
-## Support for NULL values
+## Parameter values
```

Comment:
Why did you change the title of this section? It seems like it is entirely about NULLs so the original title fit well, didn't it?

Reply:
I changed it mostly because of the following section, which is called `Return values` and also mostly focuses on the behavior with NULLs, while this one only described the NULLs as UDF parameters. 
It seems to me like one of these 2 titles is wrong. Maybe we should have 2 subsections of the NULLs section, or maybe it would be better if we had a comprehensive section on NULLs, and put the parts that aren't NULL-specific somewhere else